openapi/stripe/Ballerina.toml

```diff
@@ -6,7 +6,7 @@ name = "stripe"
 icon = "icon.png"
 distribution = "2201.2.1"
 repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
-version = "1.4.1"
+version = "1.4.2"
```

Comment:
Shall we bump to 1.5.0?

Last release was done in 2201.2.1. But now, we'll do in 2201.3.1.

Reply:
Yep - Let's do that. +1